starknet/src/voting_strategies/eth_balance_of.cairo

```diff
@@ -37,7 +37,7 @@ mod EthBalanceOfVotingStrategy {
 
             // Decode params 
             let mut params = params;
-            let (l1_account_address, slot_index) = Serde::<(
+            let (l1_token_address, slot_index) = Serde::<(
```

Comment:
this can be used to prove a slot in any contract, so token is potentially not correct. The reason I used account as thats whats they are referred to in the EVM spec, and as this contract is dealing with evm internals it made sense to follow that as close as possible. I can see how this could be confused with starknet accounts though, so `l1_contract_address` probably makes the most sense.

Reply:
Should we reconsider the wording of `balance` then maybe? Because balance seems to refer to a token (erc20 or nft). wdyt?

Reply:
oh got confused thought we were looking at `single_slot_proof.cairo` library which does indeed use `l1_contract_address` already. https://github.com/snapshot-labs/sx-starknet/blob/4aa949e68cb02f2b38454e705285ce70db3ea80f/starknet/src/utils/single_slot_proof.cairo#L31

This wrapper voting strategy i intended to make more specific to tokens hence calling it `eth_balance_of` even though it can technically be used to prove any slot as its easier to understand for people not super familiar with storage proofs etc.

Reply:
> Should we reconsider the wording of `balance` then maybe? Because balance seems to refer to a token (erc20 or nft). wdyt?

It may not be a "balance" per say, storage proof can be used to beyond token / NFT holding

Reply:
We shouldn't use token IMO, naming should be specific to what the code is doing. For users we can always name things differently in the UI, slot value is the one making more sense to me (would remove L1 as it may not even be L1 but others L2 in future).

Reply:
Decided to rename it to `EvmSlotValueVotingStrategy`, as it is the one that makes most sense. I added, as a note, that this is often used for computing a L1 token balance, or NFT balance :) 0b563e2c9f672b3afeb838bb1bac417dbac25fb4